src/microbe_stage/MicrobeStage.cs

```diff
@@ -291,6 +303,37 @@ public void StartMusic()
         Jukebox.Instance.Resume();
     }
 
+    /// <summary>
+    ///   Updates CompoundsAtMouse and MicrobesAtMouse
+    /// </summary>
+    public void UpdateMouseHover()
+    {
+        CompoundsAtMouse = Clouds.GetAllAvailableAt(Camera.CursorWorldPos);
+
+        var microbes = GetTree().GetNodesInGroup(Constants.AI_TAG_MICROBE);
+
+        if (MicrobesAtMouse != null)
+        {
+            foreach (var microbe in MicrobesAtMouse)
+                microbe.IsHoveredOver = false;
+        }
+
+        MicrobesAtMouse = new List<Microbe>();
```

Comment:
I think `MicrobesAtMouse` should be initialized inline and then just call `Clear()` here.